### PR TITLE
Finished adding the ability to add headers during auth attempts

### PR DESCRIPTION
**Pull request by Graph-X**
The yaml schema already allowed for auth > headers:, but the python did not have the functionality included. Updated the teleopti_wfm.yml as well becuase some servers will return 500 error if a content-type header does not specify application/json.  This may be the case with other apps moving forward so I figured I would finish what looked like a near completion of that capability

### CODE REVIEW

File: changeme.py [hunk 1]
```diff
@@ -603,7 +610,7 @@ def build_target_list(targets, creds, name, category):
             fp = Fingerprint(c['name'], fp=c['fingerprint'])
 
             for path in fp.urls:
-                url = '%s://%s:%s%s' % (proto, target, port, path)
```
Comment (ztgrace) [inline changeme.py]: Not sure why the change here, but only having three format strings and four arguments causes a TypeError.

**Comment by ztgrace** [comment]
Looks like some of the unit tests are failing. Make sure `nosetests -v -s --with-coverage --cover-erase --cover-package=changeme tests/*.py"` runs successfully. I've added that line to my `.git/hooks/pre-commit` file to make sure I don't break anything with my changes.

File: changeme.py [hunk 2]
```diff
@@ -372,11 +371,20 @@ def render_creds(candidate, csrf):
 def check_http(req, session, candidate, config, sessionid=False, csrf=False):
     matches = list()
     data = None
+    headers = dict()
 
     url = get_base_url(req)
     logger.debug('[check_http] base url: %s' % url)
     urls = candidate['auth']['url']
-
+    if candidate['auth']['headers']:
```
Comment (Graph-X) [inline changeme.py]: I only want to to build the headers if there's additional headers in the yml.  Wouldn't that return true no matter if those headers are present or not?  I'm looking for it to return false and move to the else where it just adds the useragent to the headers.

**Comment by ztgrace** [comment]
Thanks for the PR! Still had some build errors, but fixed them with this commit: https://github.com/ztgrace/changeme/commit/d6c036e75f83caa4cfcaea2f478fa16f02fa1a67

**Comment by Graph-X** [comment]
Glad I could help. I see what you mean about the key errors now too. Should have some new creds for you soon too.
